code/modules/transport/pods/secondary_system.dm

```diff
@@ -187,7 +187,7 @@
 			if (length(load) == 1)
 				crate = load[1]
 			else
-				crate = input(usr, "Choose which cargo to unload..", "Choose cargo")  as null|anything in load
+				crate = src.get_unloadable(usr)
```

Comment:
the `use()` proc should really just be passing `activate()` its `user` arg. `usr` should be avoided outside of unavoidable junk like `topic()`

Reply:
Agreed it works for this specific case, but parent activate() proc doesn't have any parameters, and there's more places than just in the use() proc where activate() is called (one being /obj/machinery/vehicle/Topic())

Reply:
It's "fine" for pod code and refactoring it would be a massive pain so I'd say don't scope creep the PR here.